karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
```

Comment:
Earlier we have our gulpfile and other frontend dependencies in frontend folder but after we changed it to the root directory because we want to run all the commands from the root and in our case as we are using Frontend as a separate directory we don't want to make contributors to go inside to run the Frontend tasks. All should run from the root directory. So here it's not a problem and is reasonable as we want all the configuration files in root directory. I hope you got our point why we are using them in root dir.